__tests__/connectWaiter.test.js

```diff
@@ -153,25 +151,23 @@ describe('(Redux Waiter) Connect Waiter', () => {
       dones: true,
     }
 
+    const ErrorComponent = () => <div>Hello</div>
     const Component = connectWaiter({
       name: 'mywaiter',
-      requestCreator: () => new Promise((resolve) => {
-        setTimeout(() => resolve(MOCK_RESULT), 1000)
-      }),
-      requestOnMountParams: () => (MOCK_RESULT),
-    })(function Component() {
-      return (
-        <div>Hello</div>
-      )
-    })
```

Comment:
Eslint didn't like the non-arrow function usage here. Instead of disabling the rule, I did this. I'm fine with either really.

Reply:
Probably doesn't help that the composed connectWaiter variable is named Component either.